sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/NestedColumnAliasingSuite.scala

```diff
@@ -292,7 +292,7 @@ class NestedColumnAliasingSuite extends SchemaPruningTest {
     comparePlans(optimized, expected)
   }
 
-  test("Nested field pruning for Project and Generate") {
+  test("Nested field pruning for Project and Explode") {
```

Comment:
This is unnecessary change. Can you revert it back?

Reply:
Done.